rollup/fees/rollup_fee_test.go

```diff
@@ -7,14 +7,27 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestCalculateEncodedL1DataFee(t *testing.T) {
+func TestL1DataFeeBeforeCurie(t *testing.T) {
```

Comment:
We can add back unit tests under `/rollup` in CI, and some of the 1559 unit tests (in future PRs).

Reply:
Added in [70d5402](https://github.com/scroll-tech/go-ethereum/pull/755/commits/70d54023383b424ed5c32792a8f557e394d70d84).